operator-common/src/main/java/io/strimzi/operator/common/operator/resource/CrdOperator.java

```diff
@@ -49,6 +50,8 @@ public CrdOperator(Vertx vertx, C client, Class<T> cls, Class<L> listCls, Class<
 
         if (cls.equals(Kafka.class)) {
             this.plural = Kafka.RESOURCE_PLURAL;
+        } else if (cls.equals(KafkaUser.class)) {
```

Comment:
A stupid question - but why do we have these only for Kafka and KafkaUser?

Reply:
It is not a stupid question. There should be `switch` or something more sophisticated.